docs/semantics.md

```diff
@@ -377,3 +377,38 @@ Not all of the usual file system features are supported. Most prominently:
 - File and directory permissions and ownership cannot be changed. See the permissions section above.
 - Modification times are not tracked for any inodes except for files.
 - No other times besides modification time are tracked. For example, ctime and atime are not tracked (but will be set to something reasonable). Requests to change them will appear to succeed, but the results are unspecified.
+
+# Incompatibilities with GCS (Google Cloud Storage)
```

Comment:
Is this heading needed? It can be confusing or misleading as it mentions GCS. The below "Unsupported Objects" is a good title already.

Reply:
The reason I added it was to differentiate it from the above main section `Non-standard filesystem behaviors` which is about what is supported in linux FS, but not in gcsfuse. The unsupported objects section is somewhat different, it's about what's supported in GCS, but not in gcsfuse. I can merge the change this top-level heading to `Unsupported GCS objects` and remove the lower heading `Unsupported objects`. Would that work for you?
@charith87 .

Reply:
cc @marcoa6 as this is a user-visible documentation

Reply:
Yea seems fine - Unsupported GCS objects. Will wait for marco

Reply:
I think you should just put this under line 345, 'Name conflicts'. You can add a sub section called 'Unsupported object names'